src/libs/ajax.js

```diff
@@ -66,11 +66,7 @@ const withRetryOnError = _.curry(wrappedFetch => async (...args) => {
     try {
       return await Utils.withDelay(until, wrappedFetch)(...args)
     } catch (error) {
-      // requesterPaysError may be set on responses from requests to the GCS API that are wrapped in withRequesterPays.
-      // requesterPaysError is true if the request requires a user project for billing the request to. Such errors
-      // are not transient and the request should not be retried.
-
-      const shouldNotRetry = Boolean(error.requesterPaysError)
+      const shouldNotRetry = _.intersection(_.keys(error), errorsToIgnore).length > 0
```

Comment:
It's a minor point since the collections are so small, but is there a lodash way to do a filter find first? Since you don't need a full intersection, you only need to know if at least one element of X is in Y.

Reply:
I think this is no longer needed with @nawatts' approach.

Reply:
> It's a minor point since the collections are so small, but is there a lodash way to do a filter find first? Since you don't need a full intersection, you only need to know if at least one element of X is in Y.

Good thinking. I know it's not needed anymore but for future reference Lodash's [findIndex](https://lodash.com/docs/4.17.15#findIndex) or [find](https://lodash.com/docs/4.17.15#find) could be used for this purpose.